schema.prisma

```diff
@@ -90,47 +65,31 @@ model CampaignType {
   id          String         @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
   name        String         @db.VarChar(50)
   slug        String         @unique @db.VarChar(50)
-  parentId    String?        @map("parent_id") @db.Uuid
   description String?        @db.VarChar(200)
-  campaigns   Campaign[]
+  parentId    String?        @map("parent_id") @db.Uuid
   parent      CampaignType?  @relation("ParentCategory", fields: [parentId], references: [id])
   children    CampaignType[] @relation("ParentCategory")
+  campaigns   Campaign[]
 
   @@map("campaign_types")
 }
 
-enum CampaignState {
-  initial
-  draft
-  pendingvalidation
-  approved
-  rejected
-  active
-  activependingvalidation
-  suspended
-  complete
-  disabled
-  error
-
-  @@map("campaign_state")
-}
-
 model Campaign {
   id             String        @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  state          CampaignState @default(initial)
+  state          CampaignState @default(draft)
   slug           String        @unique @db.VarChar(250)
   title          String        @db.VarChar(200)
-  excerpt        String        @db.VarChar(500)
+  essence        String        @db.VarChar(500)
   coordinatorId  String        @map("coordinator_id") @db.Uuid
   beneficiaryId  String        @map("beneficiary_id") @db.Uuid
   campaignTypeId String        @map("campaign_type_id") @db.Uuid
-  description    String?       @map("description")
+  description    String?
   targetAmount   Decimal?      @map("target_amount") @db.Decimal
-  currency       String?       @default("BGN")
+  currency       String?       @default("BGN") @db.VarChar(3)
   startDate      DateTime?     @map("start_date") @db.Timestamptz(6)
   endDate        DateTime?     @map("end_date") @db.Timestamptz(6)
   createdAt      DateTime      @default(now()) @map("created_at") @db.Timestamptz(6)
-  updatedAt      DateTime?     @updatedAt @map("updated_at") @db.Timestamptz(6)
+  updatedAt      DateTime?     @map("updated_at") @db.Timestamptz(6)
```

Comment:
It's better to mark the field with `@updatedAt` annotation add will tell Prisma to automatically update the value on update queries.

Read more at https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/schema-incompatibilities-postgres#updatedat-isnt-represented-in-database

Reply:
fully agree - now added @updatedAt annotation to all updatedAt fields in schema.prisma